authentik/blueprints/v1/common.py

```diff
@@ -219,7 +226,15 @@ def __init__(self, loader: "BlueprintLoader", node: SequenceNode) -> None:
     def resolve(self, entry: BlueprintEntry, blueprint: Blueprint) -> Any:
         query = Q()
         for cond in self.conditions:
-            query &= Q(**{cond[0]: cond[1]})
+            if isinstance(cond[0], YAMLTag):
```

Comment:
I think naming it `query_key` and `query_value` is a bit cleaner than c_0 and c_1

Reply:
Resolved